src/org/opendatakit/briefcase/model/ServerConnectionInfo.java

```diff
@@ -23,7 +23,6 @@
 public class ServerConnectionInfo {
 
   private String url;
-  private final String token; // for legacy ODK Aggregate 0.9.8 access
   private final String username;
   private final char[] password;
   private final boolean isOpenRosaServer;
```

Comment:
If you eliminate this field, which is now always true, you will discover another rich vein of code to delete. Keep going! :-)

Reply:
Ok @dcbriccetti . Will look into it right away. Thanks

Reply:
@icemc Will you be sending in an updated PR?